frontend/src/components/SignUp.jsx

```diff
@@ -1,40 +1,44 @@
 import React from "react";
+import { Formik, Form, Field } from "formik";
 
 const SignUp = () => {
-  const handleSubmit = (evt) => {
-    evt.preventDefault();
-    console.log("Form submitted");
-  };
-
   return (
     <>
```

Comment:
We can remove this React Fragment.

Reply:
Oh indeed! Somehow it didn't occur to me that the `Formik` tag wraps everything into one single element 😅